src/server/api/controllers/port.controller.js

```diff
@@ -34,9 +34,22 @@ const createPort = async ({ body }) => {
   if (ports.length !== 0) {
     throw new HttpError("Bad request", "port already exists!", 409);
   } else {
-    return knex("ports").insert({
-      name: name
-    });
+    return knex("ports")
+      .insert({
+        name: name
+      })
+      .then(([portID]) => {
+        console.log("Waypoint:", waypoint);
+        console.log("Waypoint:", waypoint.longitude, waypoint.latitude);
```

Comment:
remove console.log

Reply:
Cannot resolve this changes as this file was modified in a separate branch currently going through review cycle (see create_port_waypoint)